sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LiteralSuffixUpperCase.cs

```diff
@@ -28,7 +28,7 @@ public void Test(long ui)
         public void TestOk()
         {
             const uint a = 0U;
-            const long b = 0L;
+            const long b = 100L;
```

Comment:
was this change needed?

Reply:
it wasn't needed, however the test case has only 1-digit integer numbers, so I wanted to add a multi-digit test case